db/seed.js

```diff
@@ -58,39 +69,27 @@ const seedReviews = () => db.Promise.map([
 
 
 const seedCarts = () => db.Promise.map([
-  {contents: [{
-      product: {
-        id: 1,
-        name: 'Penguin Popper Ale',
-        price: 19.95,
-        seller_id: 1},
-      unit: 'growler',
-      quantity: 4
-    }],
+  { session: '1234',
+    lastEdited: '2015-02-09',
+    status: 'Purchased',
     user_id: 3},
   ], cart => db.model('carts').create(cart));
 
 
 const seedOrders = () => db.Promise.map([
-  {contents: [{
-      product: {
-        id: 1,
-        name: 'Penguin Popper Ale',
-        price: 19.95,
-        seller_id: 1},
-      unit: 'growler',
-      quantity: 1
-    }],
-    user_id: 3},
+  { date: '2015-02-09',
+    mailedOn: '2015-02-10',
+    status: 'Completed',
+    cart_id: 1},
   ], order => db.model('orders').create(order));
 
 
 db.didSync
   .then(() => db.sync({force: true}))
   .then(() => Promise.all([seedUsers(), seedAddresses(), seedBrewTypes(), seedUnits()]))
   .then(() => Promise.all([seedSellers()]))
-  .then(() => Promise.all([seedProducts()]))
-  .then(() => Promise.all([seedPhotos(), seedReviews(), seedCarts(), seedOrders()]))
+  .then(() => Promise.all([seedProducts(), seedInventories()]))
```

Comment:
should seedInventories be in the next .then() since it is dependent upon a Product already existing?